src/fixes/cau/loginDivider.tsx

```diff
@@ -0,0 +1,10 @@
+import { ready } from '@/DOM';
+
+if (window.location.pathname.startsWith('/login/')) {
+    await ready();
+    document.querySelectorAll<HTMLParagraphElement>('p').forEach(p => {
+        if (/_{5,}/u.exec(p.innerText)) {
```

Comment:
As we don't work with the result other than checking its truthiness, please use [`RegExp.prototype.test`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) instead of [`RegExp.prototype.exec`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec) here.